rust/module.rs

```diff
@@ -171,7 +169,14 @@ fn __build_modinfo_string_base(
 }
 
 fn __build_modinfo_string_variable(module: &str, field: &str) -> String {
-    format!("__{module}_{field}", module = module, field = field)
+    static COUNTER: AtomicUsize = AtomicUsize::new(0);
+    COUNTER.fetch_add(1, Ordering::SeqCst);
+    format!(
+        "__{module}_{field}{counter}",
+        module = module,
+        field = field,
+        counter = COUNTER.load(Ordering::SeqCst)
```

Comment:
```suggestion
        counter = COUNTER.fetch_add(1, Ordering::SeqCst)
```
`fetch_add` returns the old value.

Reply:
:+1: good point. that would increase readability

Reply:
In the current version this isn't needed.